nix/pkgs/wire-binaries.nix

```diff
@@ -13,10 +13,10 @@ let
   cni_version = "v0.9.0";
   calico_version = "v3.16.6";
 
-
+  # Note: to generate a checksum manually, run: nix-hash --flat --base32 --type sha256 elasticsearch-oss-6.8.23.deb
```

Comment:
You don't necessarily have control over all the fetchers, and what filename they come up with, so this might produce a wrong hash if the filename is different. Also, Nix is slowly moving to SRI hashes, meaning another hash format will be used.

Usually it's much easier, and reliable to do the TOFU dance, which is bumping the version alongside with replacing the hash with an all-zeroes (or known-wrong in general) hash, and then replacing it with the real hash encountered from the error message.

See https://nixos.wiki/wiki/Overlays#Overriding_a_version for a longer description.

However, this is generic Nix knowledge, and probably shouldn't be documented in `nix/pkgs/wire-binaries.nix`.